src/screens/SignIn/SignIn.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { useState } from 'react'
+import { useDispatch } from 'react-redux'
+import {
+  View,
+  Text,
+  ScrollView,
+  useWindowDimensions,
+} from 'react-native'
+import { signIn } from 'src/store/actions'
+import { StackNavigationProp } from '@react-navigation/stack'
+import { NavigationScreens } from 'src/common/enums'
+import { SignInValidationSchema } from './validation-schema'
+import { styles } from './styles'
+import { Heading, Input, Button, ButtonType, Link } from 'src/components'
+
+type RootStackParamList = {
+  Home: undefined
+  SignIn: undefined
+}
+
+type SignInScreenNavigationProp = StackNavigationProp<
+  RootStackParamList,
+  'SignIn'
+>
+
+type Props = {
+  navigation: SignInScreenNavigationProp
+}
+
+const SignIn: React.FC<Props> = ({ navigation }) => {
+  const [email, setEmail] = useState('')
+  const [password, setPassword] = useState('')
+  const dispatch = useDispatch()
+
+  const windowHeight = useWindowDimensions().height;
+
+  const handleSignInSubmit = (): void => {
+    SignInValidationSchema.validate({ email, password })
+      .then(
+        function (payload) {
+          dispatch(signIn(payload))
+          navigation.replace(NavigationScreens.HOME)
+        }
+      )
+      .catch(
+        function (err) {
+            
```

Comment:
Empty function looks strange. I think we should delete 'catch' block.

Reply:
This is blank function i planning to use for notification users with toast popupper. It can be deleted, but then we have to restore it.

Reply:
Nice. We have had notification service already. You can try to implement popupper.